hiddensearchwithrecyclerview/src/main/java/com/nicolettilu/hiddensearchwithrecyclerview/HiddenSearchWithRecyclerView.kt

```diff
@@ -396,7 +396,9 @@ class HiddenSearchWithRecyclerView @JvmOverloads constructor(
         val totalMovement = searchBarLinearLayout.height
         val remainingMovement = -searchBarLinearLayout.y.toInt()
 
-        val duration = if (totalMovement != 0) (ANIM_DURATION / totalMovement * remainingMovement) else 0
+        val duration = if(totalMovement != 0 && remainingMovement != 0) (ANIM_DURATION /
+                totalMovement * remainingMovement)
+        else 0
```

Comment:
Can you format this again? 
```kotlin
val duration = if (totalMovement != 0 && remainingMovement != 0) {
    ANIM_DURATION / totalMovement * remainingMovement 
} else {
    0
}
```

Reply:
@Bluestart83 I won't merge this until you format the code as asked.

Reply:
Done right? I'm not familiar with Github Pull request...